### ISSUE TITLE
Card Creator UX Update

### ISSUE THREAD

**Issue by AnnyBuh** [DecentralCardGame/frontend#63]
Updated order of the card creator, based on feedback from users.

**Comment by AnnyBuh** [comment]
Requirements: Feedback from Jannik & Noah about UX.

**Comment by lxgr-linux** [comment]
> Updated order of the card creator, based on feedback from users.

What is the users feedback?

**Comment by AnnyBuh** [comment]
I don't know, this issue is from half a year ago, not sure if this is still relevant. 

But there are some things that need to be changed, for example, the selection of the class and card type should be at the first step. If you want to work on this now, let's have a call about it. You can see the structural changes roughly lined out here: https://www.figma.com/file/xbMDLqz6rSN0qJjiafxe8a/UX-Crowd-Control-Website-2.0?node-id=206%3A11

**Comment by patrickwieth** [comment]
![image](https://user-images.githubusercontent.com/10943178/177029203-02bf1bfe-fa59-4e06-a8e2-517537caf4b4.png)

The problem with this is that the stuff is currently grouped according to what belongs together. It can be of course grouped differently, but in general the advice in UX is to not put more stuff on forms but rather put less stuff per page/question, because overloading the user is not good, rather guide the user through more steps than having a few big steps. This problem often arises because the people who work on such stuff have seen it 100 times already and prefer the advantage of overview and having a lot of stuff at the same time, because for them it is not overwhelming. 

Another question:
the wireframes on figma only show the first cardcreator step or am I overwhelmed and can't find the other tabs?

**Comment by AnnyBuh** [comment]
can we please not start discussions on GitHub comment section and have a call about this instead?

**Comment by patrickwieth** [comment]
yes

**Comment by patrickwieth** [comment]
here should be list of desired changes (can be discussed in a call, but must end here as a post) or this issue should be closed @AnnyBuh

**Comment by AnnyBuh** [comment]
yes I will create a wireframe. was just waiting for resources to free up from coders.

**Comment by AnnyBuh** [comment]
https://www.figma.com/file/xbMDLqz6rSN0qJjiafxe8a/UX-Crowd-Control-Website-2.0?node-id=1%3A7&t=YxbjueBzZcNhcbUZ-1

@patrickwieth please see the UX changes for the card creator

**Comment by patrickwieth** [comment]
some questions arise:
- the picture upload has a box with picture preview, how should that be arranged?
- there is a selection for fullart or framed, should it be under the preview?
- what is "Summary (Advanced Card Info)"? Is this the title of that sub page and if yes, why do the other sub pages do not have a title
- why do we display the owner, this will always be your own address? 
- what is the purpose of status? The only status that can be seen here is "prototype"? all other statuses mean the card cannot be opened in the card creator
- for the last UX update of the card creator, it was important to have an indication how much has been filled out already, see this:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/10943178/211040215-c9f63df1-4ffe-46c9-87db-c16a7b48dae2.png">
this is no longer important?

**Comment by AnnyBuh** [comment]
> some questions arise:
> 
> * the picture upload has a box with picture preview, how should that be arranged?

Picture upload is still on the first page (as it says on there)

> * there is a selection for fullart or framed, should it be under the preview?

We will remove framed option, the complications that it brings with translating the crop into for example on board cards is too big and the reward and usage of framed cards is too small. The problems of readability with using only fullart cards did not occur (as I predicted). 

> * what is "Summary (Advanced Card Info)"? Is this the title of that sub page and if yes, why do the other sub pages do not have a title

Yes this is the title, it should look like the Advanced Card Information page titles. You are right, this can be in this bar that you progress bar that you added. It can be kept as "Summary and Publish".

> * why do we display the owner, this will always be your own address?

I added it to make it congruent with the Advanced Card Information page. 

> * what is the purpose of status? The only status that can be seen here is "prototype"? all other statuses mean the card cannot be opened in the card creator

I added it to make it congruent with the Advanced Card Information page. 

> * for the last UX update of the card creator, it was important to have an indication how much has been filled out already, see this:
> 
> <img alt="image" width="689" src="https://user-images.githubusercontent.com/10943178/211040215-c9f63df1-4ffe-46c9-87db-c16a7b48dae2.png">
> 
> this is no longer important?

We should keep it, I might do some visual redesign with the whole page redesign.

**Comment by patrickwieth** [comment]
>  The problems of readability with using only fullart cards did not occur (as I predicted).

It did occur, reconstruction was pushed to the chain with framed because of this 
<img width="269" alt="image" src="https://user-images.githubusercontent.com/10943178/211200915-c5a3908b-50fd-4460-a799-b43df79fdc9c.png">
But still we should remove it, since very rare and can be solved easily by the artist

> I added it to make it congruent with the Advanced Card Information page.

Ok, so shall we also add the other information from the Advanced Card Info? There is more stuff that is totally useless in this context, for example "Votepool: 0credits" or "Latest Voting Results", Artist though makes sense and should be possible to set somewhere in the card creator. We also need some kind of card creator for the artist, where only the artwork can be set.

**Comment by AnnyBuh** [comment]
I think it makes most sense to add all the info from Advanced Card Info and just let it unfiltered. Yes when it comes to spitting the card creator for artist view and game mechanics view we have to find a solution. For now would it be possible to have 2 people access 1 card prototype (pre-publish but saved somehow)? That would already help.

**Comment by patrickwieth** [comment]
It is possible to build the website so that the artist can also edit the card. A good solution would imo be to remove the image upload if the editor is not the artist and remove the card content stuff if the editor is the artist. We need a field to set an artist though.

Regarding the data from Advanced Card Info that makes sense to be displayed, even if nonsensical: Can you share your insight, why it makes sense to display 7 data fields that are either not defined at this time (5) or will always display the same value (2).

**Comment by AnnyBuh** [comment]
My only argument for always displaying all the information is to be as transparent about all the factors that will be of importance for later stages of the card as possible. Other than that I don't really think it makes sense.
